cli/lib/exec/shared.js

```diff
@@ -58,8 +59,52 @@ const checkConfigFile = (options) => {
   }
 }
 
+/**
+* Checks if option is an empty string and throws an error if it is
+* @param {Object} options - options object
+* @param {string} optionName - name of the option to check
+* @returns {void}
+* */
+const checkNonEmptyOption = (options, optionName) => {
+  if (typeof options[optionName] === 'string' && options[optionName].trim().length === 0) {
+    const { shortFlag, flag } = config[optionName]
+
+    throwInvalidOptionError(errors.emptyNonEmptyOption(flag ? flag : shortFlag))
+  }
+}
+
+/**
+* Validates all non-empty options in the options object
+* @param {Object} options - options object
+* @returns {void}
+* */
+const validateNonEmptyOptions = (options) => {
+  const nonEmptyOptions = [
```

Comment:
Suggest moving this list into `options.js`. May make sense to move the entire function there as well.